tools/NuGet/DynamoVisualProgramming.ZeroTouchLibrary/ZeroTouchLibrary.nuspec

```diff
@@ -12,5 +12,8 @@
     <summary>This package contains all that is required to get up and running building zero touch libraries for the Dynamo Visual Programming language.</summary>
     <copyright>Copyright Autodesk 2016</copyright>
     <iconUrl>https://raw.githubusercontent.com/DynamoDS/Dynamo/master/doc/distrib/Images/logo_square_32x32.png</iconUrl>
-  </metadata>
+	<dependencies>
+		<dependency id= "DynamoVisualProgramming.DynamoServices" version="1.0.0-beta1"/>
```

Comment:
Let's make this guy dependent on 1.0.0-beta2 and update DynamoServices package version to 1.0.0-beta2 we have some new changes and they have not been updated yet on NuGet.